src/platform/ESP32/InetPlatformConfig.h

```diff
@@ -41,3 +41,4 @@
 #ifndef INET_CONFIG_NUM_UDP_ENDPOINTS
 #define INET_CONFIG_NUM_UDP_ENDPOINTS CONFIG_NUM_UDP_ENDPOINTS
 #endif // INET_CONFIG_NUM_UDP_ENDPOINTS
+#define INET_CONFIG_ENABLE_IPV4 1
```

Comment:
Why is this required? For the OTA examples?

Reply:
I added this to make function `Inet::IPAddress::FromString` work. It is a function transfering an IP String to `Inet::IPAddress`.

Reply:
This will have to be reverted - you are not  allowed to assume IPV4 is enabled.

Reply:
#12700 reverts this. Builds with IPV4 off have to be valid in Matter (SPEC requires IPV6 only and IPV6 is mandatory for thread compatibility)